src/pages/Welcome/welcomeStyles.css

```diff
@@ -0,0 +1,16 @@
+.welcomeScreen {
+  background-image: url('./../../welcomeBackground.png');
+  background-size: cover;
```

Comment:
You could have a more flexible option here with `background: linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.17) 28.26%, rgba(255, 255, 255, 0) 100%), #29620F;` (you can find this in Figma's Inspect tab)

Reply:
If you did that, you probably wouldn't have to apply absolute positioning to .welcomeScreen

Reply:
Oh nice! Thanks, I'll update the background tomorrow morning! I think I might still have to keep the absolute positioning, because that's the only way I could find to make the div fill the whole App.js, but I'll apply the gradient and play around with it a bit!

Reply:
Replaced the background image with the gradient, which is definitely a better solution, thanks @dariatsvetkova ! Had to keep the `position: absolute` though, otherwise the gradient wouldn't fill the full App.js:
![image](https://user-images.githubusercontent.com/58532369/120071054-6bea3480-c05b-11eb-9020-2c3da5740b7b.png)